extensions/vscode/src/commands/new-route.ts

```diff
@@ -49,8 +50,21 @@ export const newRoute = async (uri: Uri | undefined): Promise<void> => {
   if (uri === undefined) {
     selectedPath = resolveDartFrogProjectPathFromActiveTextEditor();
 
-    if (selectedPath === undefined) {
-      selectedPath = resolveDartFrogProjectPathFromWorkspaceFolders();
+    if (!selectedPath) {
```

Comment:
This snippet, and the above are identical if I am not mistaken, can we wrap it in a method or function so we don't repeat ourselves?

Reply:
This is a valid suggestion. I do plan to refactor to reuse logic, if you don't mind I'll do the refactoring in another Pull Request 🙌

Reply:
I've simplified the logic, let me know what you think now 🙌

Reply:
I've simplified the logic, let me know what you think now 🙌